packages/okta-angular/README.md

```diff
@@ -30,7 +30,11 @@ This library is tested against the latest version of Angular (currently 6), and
 This library is available through [npm](https://www.npmjs.com/package/@okta/okta-angular). To install it, simply add it to your project:
 
 ```bash
+# npm
```

Comment:
This is closer to what I was talking about in the other comment (especially with the yarn part only mentioned in the development section)